templates/PlayFab_Api.cpp.ejs

```diff
@@ -29,17 +31,15 @@ namespace PlayFab
 <% for (var callIdx = 0; callIdx < api.calls.length; callIdx++) { var apiCall = api.calls[callIdx]; %>
     void PlayFab<%- api.name %>API::<%- apiCall.name %>(
         <%- apiCall.request %>& request,
-        ProcessApiCallback<<%- apiCall.result %>> callback,
-        ErrorCallback errorCallback,
+        const ProcessApiCallback<<%- apiCall.result %>>& callback,
```

Comment:
Oo, make sure these are non const&, you may want to double check that you merge with master

Reply:
Ok, I think const is right, but Deb and Victor convinced me that & is bad.